tests/components/gree/test_climate.py

```diff
@@ -399,6 +403,9 @@ async def test_send_target_temperature(hass, discovery, device, units, temperatu
     state = hass.states.get(ENTITY_ID)
     assert state is not None
     assert state.attributes.get(ATTR_TEMPERATURE) == temperature
+    assert state.attributes.get(ATTR_CURRENT_TEMPERATURE) != state.attributes.get(
+        ATTR_TEMPERATURE
+    )
```

Comment:
This is not testing the expected value, please adjust it to compare against the expected value (instead of the not case).